Package.swift

```diff
@@ -6,25 +6,34 @@ import PackageDescription
 let package = Package(
     name: "Tiqr",
     defaultLocalization: "en",
-    platforms: [.iOS(.v8)],
+    platforms: [.iOS(.v13)],
     products: [
         .library(
             name: "Tiqr",
             targets: ["Tiqr"]),
     ],
+    dependencies: [
+        .package(url: "https://github.com/roberthein/TinyConstraints.git", from: "4.0.0"),
+    ],
     targets: [
         .target(
             name: "Tiqr",
-            dependencies: ["TiqrCoreObjC", "TiqrCore"]),
+            dependencies: ["TiqrCoreObjC", "TiqrCore"]
+        ),
         .target(
             name: "TiqrCore"
         ),
         .target(
             name: "TiqrCoreObjC",
-            dependencies: ["TiqrCore"],
+            dependencies: ["TiqrCore", "EduIDExpansion"],
             resources: [
                 .process("Resources/Audio/cowbell.wav"),
-                .process("Resources/Views/HTML/start.html")
-            ])
+                .process("Resources/Views/HTML/start.html")]
+        ),
+        .target(
+            name: "EduIDExpansion",
+            dependencies: ["TinyConstraints"],
+            resources: [.process("Resources/Assets.xcassets")]
```

Comment:
FYI: xcassets should automatically work. No need to list those afaik.

Reply:
true I will remove, though always you need to use Bundle.module for Colors as well as images afaik